src/OpenTelemetry.Exporter.Prometheus/OpenTelemetry.Exporter.Prometheus.csproj

```diff
@@ -1,12 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net46;netstandard2.0;</TargetFrameworks>
+    <TargetFrameworks>net452;net46;netstandard2.0;</TargetFrameworks>
```

Comment:
I start to wonder maybe we can extract this into a common props file? Maybe next PR 😃